src/js/routes/development.js

```diff
@@ -0,0 +1,36 @@
+import React, { lazy, Suspense } from "react";
+
+const GraphiQL = lazy(() =>
+  import(/* webpackChunkName: "graphiql" */ "#SRC/js/components/GraphiQL")
+);
+
+function GraphiQLLoader() {
+  return <div />;
+}
+
+function GraphiQLWrapper() {
+  return (
+    <div style={{ width: "100%", backgroundColor: "#FFF" }}>
+      <Suspense fallback={GraphiQLLoader}>
+        <GraphiQL />
+      </Suspense>
+    </div>
+  );
+}
+
+GraphiQLWrapper.routeConfig = {
+  label: "GraphiQL",
+  matches: /^\/development\/graphiql/
+};
+
+export default function developmentFactory(
```

Comment:
this does not look like a factory to me. 🤔

i'd even prefer an anonymous function here over `developmentFactory`.

Reply:
👍